.github/label-commenter-config.yml

```diff
@@ -16,18 +16,37 @@ labels:
         action: open
-      pr: 
+      pr:
         body: Checking in... it has been awhile since we&#39;ve heard from you on this issue. Are you still working on it? Please let us know and please don&#39;t hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
         action: open
   - name: issue/dco
     labeled:
       issue:
         body: "🚨 Alert! Git Police! We couldn’t help but notice that one or more of your commits is missing a sign-off. _A what?_ A commit sign-off (your email address).\n\n
-          To amend the commits in this PR with your signoff using the instructions provided in the DCO check above. \n\n
-          To configure your dev environment to automatically signoff on your commits in the future, see [these instructions](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)."
+         To amend the commits in this PR with your signoff using the instructions provided in the DCO check. \n\n
+         To configure your dev environment to automatically signoff on your commits in the future, see [these instructions](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)."
         action: open
       pr:
         body: "🚨 Alert! Git Police! We couldn’t help but notice that one or more of your commits is missing a sign-off. _A what?_ A commit sign-off (your email address).\n\n
-          To amend the commits in this PR with your signoff using the instructions provided in the DCO check above. \n\n
-          To configure your dev environment to automatically signoff on your commits in the future, see [these instructions](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)."
+         To amend the commits in this PR with your signoff using the instructions provided in the DCO check. \n\n
+         To configure your dev environment to automatically signoff on your commits in the future, see [these instructions](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)."
+        action: open
+  - name: component/ui
+    labeled:
+      issue:
+        body: This issue has been labeled with 'component/ui'. 🧰 Here are docs on [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui). 🎨 Here is the [Meshery UI Figma File](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4%3A0) File. Lastly, here are docs on [Contributing to Meshery's End-to-End Tests Using Cypress](https://docs.meshery.io/project/contributing/contributing-cypress).
```

Comment:
Given that we're moving away from Cypress to Playwright, we'll either want to exclude this link in the comment now, or include it, but create an issue to track the need to updated it when the Playwright version of the contrib instructions are available. Either way, we need one or two new issues created:

1. Create a Playwright contrib doc.
2. Update this link.

@MarioArriaga92